docs/content/admin/quickstart.mdx

```diff
@@ -33,79 +33,146 @@ You can install Node from the [official website](https://nodejs.org/en/).
-git clone https://github.com/medusajs/admin medusa-admin
-cd medusa-admin
+You can add the following script to `package.json` which manually builds the admin:
+
+```json title=package.json
+{
+  "scripts": {
+    // other scripts...
+    "build:admin": "medusa-admin build"
+  }
+}
 ```
 
-Then, install the dependencies:
+Then, you can build the admin dashboard by running the following command:
 
 ```bash npm2yarn
-npm install
+npm run build:admin
 ```
 
+#### Build Options
+
+You can add the following options to the build command:
+
+- `--deployment`: a boolean value indicating that the build should be ready for deployment. When this option is added, options are not loaded from `medusa-config.js` anymore, and it means the admin will be built to be hosted on an external host. For example, `medusa-admin build --deployment`.
+- `--backend` or `-b`: a string specifying the URL of the Medusa backend. This can be useful with the `--deployment` option. The default here is the value of the environment variable `MEDUSA_BACKEND_URL`. For example, `medusa-admin build --deployment --backend example.com`
+- `--out-dir` or `-o`: a string specifying a custom path to output the build files to. By default, it will be the `build` directory. For example, `medusa-admin --deployment --out-dir public`.
+- `--include` or `-i`: a list of strings of paths to files you want to include in the build output. It can be useful if you want to inject files that are relevant to your external hosting, such as adding a `200.html` file that is needed for redirects on Surge. For example, `medusa-admin --deployment --include 200.html`
+- `--include-dist` or `-d`: a string specifying the path to copy the files specified in `--include` to. By default, the files are coopied to the root of the build directory. You can use this option to change that. For example, `medusa-admin --deployment --include 200.html --include-dist static`.
+
+### Step 3: Test the Admin Dashboard
+
+:::tip
+
+If you disabled the `serve` option, you need to serve the admin dashboard separately as explained in [this section](#serve-the-admin-dashboard-separately).
+
+:::
+
+You can test the admin dashboard by running the following command in the directory of the Medusa backend:
+
+```bash npm2yarn
+npm run start
+```
+
+This starts the Medusa Backend and the admin dashboard. By default, the admin will be available on the URL `localhost:9000/app`. If you set the `path` option, then the admin will be available on `localhost:9000/<PATH>` with `<PATH>` being the value of the `path` option.
+
 <Feedback 
    event="survey_admin_quickstart"
    question="Did you set up the admin successfully?"
    positiveQuestion="Is there anything that should improved?"
    negativeQuestion="Please describe the issue you faced."
 />
 
-:::tip
-
-If you run into errors during the installation, check out [this troubleshooting guide](../troubleshooting/common-installation-errors.mdx).
+---
 
-:::
+## Serve the Admin Dashboard Separately
```

Comment:
Would it make sense to change this section, so it simply links to the deployment to Vercel docs, and note that they can use `medusa-admin dev` to spin up the admin in development?